TAScheduler/migrations/0001_initial.py

```diff
@@ -0,0 +1,97 @@
+# Generated by Django 4.2.7 on 2023-11-19 03:39
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Course',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('course_id', models.IntegerField()),
+                ('semester', models.CharField(max_length=11)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Instructor',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('max_assignments', models.IntegerField(default=6, validators=[django.core.validators.MaxValueValidator(6), django.core.validators.MinValueValidator(0)])),
+            ],
+        ),
+        migrations.CreateModel(
+            name='TA',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('grader_status', models.BooleanField()),
+                ('max_assignments', models.IntegerField(default=6, validators=[django.core.validators.MaxValueValidator(6), django.core.validators.MinValueValidator(0)])),
+            ],
+        ),
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('email_address', models.CharField(max_length=90)),
+                ('password', models.CharField(max_length=30)),
+                ('first_name', models.CharField(max_length=30)),
+                ('last_name', models.CharField(max_length=30)),
+                ('home_address', models.CharField(max_length=90)),
+                ('phone_number', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='TAToCourse',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('course', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='TAScheduler.course')),
+                ('ta', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='TAScheduler.ta')),
+            ],
+        ),
+        migrations.AddField(
+            model_name='ta',
+            name='user',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='TAScheduler.user'),
+        ),
+        migrations.CreateModel(
+            name='Section',
```

Comment:
We might need to discuss this over a call - My section in the database refers to a Lecture & Lab table ...  the Lecture & the Lab table are the ones that hold the foreign keys, not the section